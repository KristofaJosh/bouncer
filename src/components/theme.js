import * as colors from "../components/Colors/color";
const theme = {
  colors,
};
export default theme;