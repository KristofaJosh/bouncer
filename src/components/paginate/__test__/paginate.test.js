import React from "react";
import { render, fireEvent } from "@testing-library/react";
import theme from "../../theme";
import { ThemeProvider } from "styled-components";
import Pagination from "..";
import '@testing-library/jest-dom/extend-expect'

test("test the state of the item", async () => {
  const { getByText } = render(
    <ThemeProvider theme={theme}>
      <Pagination />
    </ThemeProvider>
  );

  const bgcolor = await getByText("2");

  expect(bgcolor.style.color).toBe('black');
  expect(bgcolor.style.color).toMatchSnapshot();
});

test('test background color of active state', () => {
    const { getByText } = render( 
    <ThemeProvider theme={theme}>
        <Pagination />
    </ThemeProvider>
    )
    const bg = getByText('1')
    expect(bg.style.backgroundColor).toBe('black')
    expect(bg.style.color).toBe('white')
    expect(bg.style.backgroundColor).toMatchSnapshot();
})
