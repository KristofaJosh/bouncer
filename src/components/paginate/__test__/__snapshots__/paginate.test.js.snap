// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test background color of active state 1`] = `"black"`;

exports[`test the state of the item 1`] = `"black"`;
